foo.go

```diff
@@ -0,0 +1,3 @@
+package foo
+
+var Public string = "hello"
```

Comment:
exported var Public should have comment or be unexported<br>should omit type string from declaration of var Public; it will be inferred from the right-hand side